docs/integrations/sources/harvest.md

```diff
@@ -78,7 +78,8 @@ The connector is restricted by the [Harvest rate limits](https://help.getharvest
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                            |
-| :------ | :--------- | :------------------------------------------------------- | :--------------------------------------------------------------------------------- |
+|:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------|
+| 0.1.12  | 2022-12-16 | [20572](https://github.com/airbytehq/airbyte/pull/20572) | Speed up SATs                                                                      |
```

Comment:
```suggestion
| 0.1.12  | 2022-12-16 | [20572](https://github.com/airbytehq/airbyte/pull/20572) | Introduce replication end date at the source level                                                                       |
```
Why is it speedin up SATs? The main change looks related to replication_end_date imo.

Reply:
Could you please add `replication_end_date` to the spec with `airbyte_hidden: true`? I'd love the SAT config to match the spec as much as possible. 😄

Reply:
done